### PR TITLE
Reduce the amount of URLs Amp consent banner E2Es run on

### PR DESCRIPTION
**Pull request by LilyL0u**
Due to timeouts in the pipeline we are seeing whether reducing the number of these AMP consent banner tests helps this.  The tests have 3 retries on them and as they are flakey these retries are used a lot, so these tests take longer than other tests.

Only run on 11/12 URLs
5 mundo
5/6 pashto (one only runs on test environment)

18 removed

Services tested are Pashto and Mundo (Persian removed)

Page types checked once:
Live radio
Tc2 map
Cps sty
Pgl
On demand audio brand
On demand tv brand (test only)
Article
Front page
Index
Most read
CPS map

**Code changes:**
Disabled and commented out URLs

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project

**Testing:**

Passes when run against live environment

### CODE REVIEW

File: cypress/support/config/settings.js [hunk 1]
```diff
@@ -5250,58 +5250,58 @@ module.exports = () => ({
...
-              '/persian/world-51497110', // CPS MAP
-              '/persian/media-49522521', // CPS MAP with live stream
-              '/persian/world/2016/06/160613_om_naked_dining', // TC2 MAP
-              '/persian/bbc_persian_radio/liveradio', // Live Radio
-              '/persian/bbc_dari_radio/liveradio', // Live Radio
-              '/persian/bbc_dari_radio/programmes/p0340v0s', // On Demand Radio Brand
-              '/persian/bbc_persian_radio/programmes/p0340vyx', // On Demand Radio Brand
-              '/persian/bbc_dari_radio/w3ct0bst', // On Demand Radio Episode
-              '/persian/bbc_persian_radio/w3ct0s49', // On Demand Radio Episode
+              // '/persian/articles/c7eel0lmr4do', // Article
```
Comment (ryanmccombe) [inline cypress/support/config/settings.js:5277]: I think just setting `enabled` to false is enough - shouldn't need to comment all these out as well

Doesn't do any harm though

**Review by ryanmccombe — APPROVED** [review]

File: cypress/support/config/settings.js (hunk 1 above)
Comment (LilyL0u) [reply cypress/support/config/settings.js:5277]: You are right, it usually does. For some reason this time it kept running the tests even though I had changed it to false! Maybe it wasn't updating correctly. I got fed up and just commented them out to make it work.

**Review by FK78 — APPROVED** [review]

**Comment by andrewscfc** [comment]
Do we still need to make this change @LilyL0u ?

**Comment by LilyL0u** [comment]
> Do we still need to make this change @LilyL0u ?

It's not really necessary anymore, but will shorten the test run slightly (these tests have retries that are used quite a lot)

**Pull request merged**